spec/lib/i18n/hygiene/variable_checker_spec.rb

```diff
@@ -3,8 +3,8 @@
 
 describe I18n::Hygiene::VariableChecker do
   let(:i18n_wrapper) { instance_double("TC::I18nWrapper") }
-  let(:checker) { I18n::Hygiene::VariableChecker.new("test_key", i18n_wrapper, [:fr]) }
-  let(:checker_markdown) { I18n::Hygiene::VariableChecker.new("test_key_markdown", i18n_wrapper, [:fr]) }
+  let(:checker) { I18n::Hygiene::VariableChecker.new("test_key", i18n_wrapper, :en, [:fr]) }
+  let(:checker_markdown) { I18n::Hygiene::VariableChecker.new("test_key_markdown", i18n_wrapper, :en, [:fr]) }
```

Comment:
I really like the explicit arguments to `I18n::Hygiene::VariableChecker` - I like that it's easier to see that it's confirming the variables in `[:fr]` match `:en`